cs/testing-with-travis.texy

```diff
@@ -0,0 +1,250 @@
+Průběžné testování s Nette Tester
+********************************
+
+/--div .[perex]
+Znáte to. Napsali jste testy, které teď procházejí, poplácali se po zádech a váš spánek začal být mnohem klidnější. Máte jistotu, že váš kód spolehlivě funguje. A pak se to stane. Hodiny ukazují 3:15 a do nočního ticha se zakousne nepříjemný zvuk drnčení mobilu na nočním stolku. Volá šéf. Zase to nefunguje. Zalije vás studený pot. Vždyť na to máme testy! V tu chvíli se vynoří vzpomínka na páteční odpoledne - spěcháte domů a zapomínáte spustit testy po poslední úpravě.
+
+Abyste předešli podobným zážitkům, ukážeme si, jak průběžně testovat pomocí populárního nástroje [Travis CI](https://travis-ci.org/):
+
+- nastavení testovacího prostředí
+- instalace závislostí pomocí Composeru
+- spouštění testů pomocí Nette/Tester
+- nastavení dalších služeb
+- [TL;DR |#Výsledný soubor]
+- integrace s GitHubem
+\--
+
+Jak funguje Travis CI
+=====================
+
+Travis CI((Continuous Integration)) (dále jen Travis) je velmi populární služba pro automatizaci [průběžné integrace](http://cs.wikipedia.org/wiki/Průběžná_integrace). Je poskytována zdarma((Pro soukromé repozitáře je zpoplatněna.)) a nabízí automatizované spouštění testů po zápisu změny do repozitáře. Je plně integrována do prostředí GitHubu.
+
+Travis se konfiguruje souborem `.travis.yml` umístěným v kořenovém adresáři projektu. Obsahuje několik bloků, které si podrobněji popíšeme.
+
```

Comment:
Ještě mě nepadá, že bych tam dodal, že většina z těch bloků je nepovinná, ať to nevypadá tak složitě.